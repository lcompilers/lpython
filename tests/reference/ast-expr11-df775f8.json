{
    "basename": "ast-expr11-df775f8",
    "cmd": "lpython --show-ast --indent --no-color {infile} -o {outfile}",
    "infile": "tests/expr11.py",
    "infile_hash": "940f2d32759315dfb8d54ea50819f2bfef9737e486615703609fd47a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-expr11-df775f8.stdout",
    "stdout_hash": "60719c0d166865f8157e6479a6bd6fc7d300b2d2eff48d8b1553fc8c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}