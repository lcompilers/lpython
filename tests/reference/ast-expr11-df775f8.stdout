(Module
    [(FunctionDef
        test_StrOp_repeat
        ([]
        []
        []
        []
        []
        []
        [])
        [(AnnAssign
            (Name
                s
                Store
            )
            (Name
                str
                Load
            )
            ()
            1
        )
        (Assign
            [(Name
                s
                Store
            )]
            (BinOp
                (ConstantStr
                    "a"
                    ()
                )
                Mult
                (ConstantInt
                    2
                    ()
                )
            )
            ()
        )
        (Assign
            [(Name
                s
                Store
            )]
            (BinOp
                (ConstantStr
                    "a"
                    ()
                )
                Mult
                (UnaryOp
                    USub
                    (ConstantInt
                        1
                        ()
                    )
                )
            )
            ()
        )
        (Assign
            [(Name
                s
                Store
            )]
            (BinOp
                (ConstantStr
                    "test"
                    ()
                )
                Mult
                (ConstantInt
                    5
                    ()
                )
            )
            ()
        )
        (Assign
            [(Name
                s
                Store
            )]
            (BinOp
                (ConstantInt
                    4
                    ()
                )
                Mult
                (ConstantStr
                    "bb"
                    ()
                )
            )
            ()
        )
        (Assign
            [(Name
                s
                Store
            )]
            (BinOp
                (UnaryOp
                    USub
                    (ConstantInt
                        40
                        ()
                    )
                )
                Mult
                (ConstantStr
                    "bb"
                    ()
                )
            )
            ()
        )
        (Assign
            [(Name
                s
                Store
            )]
            (BinOp
                (BinOp
                    (ConstantInt
                        3
                        ()
                    )
                    Mult
                    (ConstantStr
                        "a"
                        ()
                    )
                )
                Mult
                (ConstantInt
                    3
                    ()
                )
            )
            ()
        )]
        []
        ()
        ()
    )]
    []
)
